docs/memestra-cache.rst

```diff
@@ -0,0 +1,34 @@
+Memestra-cache
+==============
+
+To handle the cache, memestra installs a tool named ``memestra-cache``.
```

Comment:
Memestra uses a cache to speedup some computations. To handle it, it installs a tool named...